pkg/skaffold/runner/v1/new.go

```diff
@@ -93,7 +93,8 @@ func NewForConfig(runCtx *runcontext.RunContext) (*SkaffoldRunner, error) {
 		Syncer:   sync.NewSyncProvider(runCtx, kubectlCLI),
 	}
 
-	deployer, err = runner.GetDeployer(runCtx, provider, labeller.Labels())
+	var local bool
+	deployer, local, err = runner.GetDeployer(runCtx, provider, labeller.Labels())
```

Comment:
I think having another return target (`local`) goes against the abstraction of the `Deployer` interface implementation handling all these nuances.

Reply:
yeah you're right. this is actually just for the `failIfClusterNotReachable()` check, but that can probably get moved into the `Deployer`s. let me raise another PR